modules/security-context-constraints-psa-about.adoc

```diff
@@ -0,0 +1,71 @@
+// Module included in the following assemblies:
+//
+// * authentication/understanding-and-managing-pod-security-admission.adoc
+// * operators/operator_sdk/osdk-complying-with-psa.adoc
+
+:_content-type: CONCEPT
+[id="security-context-constraints-psa-about_{context}"]
+= About pod security admission
+
+{product-title} includes link:https://kubernetes.io/docs/concepts/security/pod-security-admission[Kubernetes pod security admission]. Pods that do not comply with the pod security admission defined globally or at the namespace level are not admitted to the cluster and cannot run.
+
+Globally, the `privileged` profile is enforced, and the `restricted` profile is used for warnings and audits.
+
+To configure the pod security admission settings at the namespace level, set a label on the namespace in the following format: `pod-security.kubernetes.io/<mode>: <profile>`, where `<mode>` is the mode to configure and `<profile>` is the profile to use for that mode.
```

Comment:
@bergerhoffer Instead of this "**format**" (or along with the format), can we add a command example like below:
`oc label namespace <namespace> pod-security.kubernetes.io/<mode>=<profile> --overwrite`

Others /lgtm from QE.

Reply:
Thanks for the suggestion. I moved this into an actual procedure module on how to set the labels. Let me know what you think!